openedx/features/content_tagging/system_defined_taxonomies/object_tags.py

```diff
@@ -0,0 +1,90 @@
+"""
+ObjectTags for System-defined Taxonomies
+"""
+from openedx_tagging.core.tagging.system_defined_taxonomies.object_tags import (
+    SystemDefinedIds as TaxonomyIds,
+    ModelObjectTag,
+    UserObjectTag,
+    LanguageObjectTag,
+)
+from openedx_tagging.core.tagging.registry import register_object_tag_class
+from organizations.models import Organization
+
+from ..models import CourseObjectTagMixin, BlockObjectTagMixin
+
+
+class SystemDefinedIds(TaxonomyIds):
+    """
+    System-defined taxonomy IDs
+    """
+    OrganizationTaxonomy = 2
+    AuthorTaxonomy = 3
+
+
+class OrganizationObjectTag(ModelObjectTag):
+    """
+    ObjectTag used on Organization system-defined taxonomy
+    """
+
+    system_defined_taxonomy_id = SystemDefinedIds.OrganizationTaxonomy.value
+
+    class Meta:
+        proxy = True
+
+    tag_class_model = Organization
+
+
+class AuthorObjcetTag(UserObjectTag):
+    """
+    ObjectTag used on Author system-defined taxonomy
+    """
+
+    system_defined_taxonomy_id = SystemDefinedIds.AuthorTaxonomy.value
+
+    class Meta:
+        proxy = True
+
+
+class OrganizationCourseObjectTag(OrganizationObjectTag, CourseObjectTagMixin):
```

Comment:
As per [this comment](https://github.com/open-craft/openedx-learning/pull/3/files#diff-2ecc1274a15be58e16fa617e6e5860e1b2815b6cbeffa200daea9676c07c4bfcR88), we should switch the inheritance order around for these subclasses, e.g.

```suggestion
class OrganizationCourseObjectTag(CourseObjectTagMixin, OrganizationObjectTag):
```

Reply:
Outdated with new changes